app/models/user.rb

```diff
@@ -5,4 +5,21 @@ class User
   field :encrypted_password, type: String
   field :auth_token, type: String
   attr_accessor :password
+  has_many :events
```

Comment:
what will happen on User.delete? is this dependent? nullify? :stuck_out_tongue_closed_eyes: